setup.cfg

```diff
@@ -1,6 +1,9 @@
 [tool:pytest]
 testpaths=tests
 
+# https://github.com/googlefonts/fontbakery/issues/2079
+filterwarnings = ignore::DeprecationWarning
```

Comment:
that's a bit too broad.. how about "ignore:fromstring:DeprecationWarning" and "ignore:tostring:DeprecationWarning"?

Reply:
oh! I agree with that!

Reply:
@madig please open a subsequent PR with those changes :-P